onnxruntime/contrib_ops/cuda/decoder/decoder_masked_self_attention.cc

```diff
@@ -70,29 +70,29 @@ Status DecoderMaskedMultiheadAttention<T1, T2>::ComputeInternal(OpKernelContext*
 
   // This kernel is for decoding only (i.e.) sequence length has to be 1
   if (sequence_length != 1) {
-    return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT, "Input sequence length should be 1 to use DecoderMaskedMultiheadAttention");
+    return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT, "Input sequence length should be 1 to use DecoderMaskedSelfAttention");
   }
 
   // TODO(hasesh): In future, we may support CrossAttention. Currently, this kernel only supports SelfAttention.
   if (parameters.sequence_length != parameters.kv_sequence_length) {
-    return ORT_MAKE_STATUS(ONNXRUNTIME, NOT_IMPLEMENTED, "DecoderMaskedMultiheadAttention only supports self attention currently");
+    return ORT_MAKE_STATUS(ONNXRUNTIME, NOT_IMPLEMENTED, "DecoderMaskedSelfAttention only supports self attention currently");
```

Comment:
Nit: Change this to ORT_ENFORCE(parameters.sequence_length == parameters.kv_sequence_length); as there are no plans to support CrossAttention in this kernel ?

Reply:
will add in next PR